airbyte-bootloader/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     implementation project(':airbyte-config:persistence')
     implementation project(':airbyte-db:lib')
     implementation project(":airbyte-json-validation")
+    implementation project(':airbyte-protocol:models')
```

Comment:
do we need this in `implementation` or only in `testImplementation`? I think it is only used in tests?

Reply:
It was needed by another import. It turns out that it was because of an unusued method. I have removed it.